### PR TITLE
Whois: Update Style #1414

### LINKED ISSUE #1414: Whois Spice - Additional Updates Needed
We need to make updates to https://github.com/duckduckgo/zeroclickinfo-spice/pull/1331. In the end, it should look like this:

screen shot 2015-01-14 at 2 14 36 pm

Changes:
- Remove byline completely ("whois lookup")
- CSS changes:

``` css
.zci--whois .zci__more-at { padding-top: 0; }
.zci--whois .record td { padding-left: 0; }
.zci--whois .record .record__cell__key { text-transform: capitalize; }
```

Finally, we need to make these changes (not shown):
- Remove colons at the end of each key
- "Expires" should be "Expires On"

And that's it!

### PR DESCRIPTION
**Pull request by MrChrisW**
Fixes #1414 @abeyang  @jagtalon 

selection_070

### CODE REVIEW

File: share/spice/whois/whois.css [hunk 1]
```diff
@@ -16,15 +16,17 @@
...
 
 .zci--whois .record td {
     padding: 0.4em;
+    padding-left: 0; 
     border-top: solid 1px #DDD;
     border-bottom: solid 1px #DDD;
 }
 
 .zci--whois .record .record__cell__key {
     width: 150px;
```
Comment (MrChrisW) [inline share/spice/whois/whois.css]: Is this `width` fine? It adds spacing between the keys and results.

**Comment by MrChrisW** [comment]
@abeyang I don't believe we can remove colons at the end of each key as this is built into the `records` template.

**Comment by abeyang** [comment]
Thanks @MrChrisW ! We'll take a look at it and get back to you.. will also look into the colons. /cc @jagtalon

**Comment by jagtalon** [comment]
@MrChrisW nice! @abeyang yup, coming right up!

**Comment by jagtalon** [comment]
@abeyang https://ddh1.duckduckgo.com/?q=whois+facebook.com&ia=whois. Looks good to me.

**Comment by moollaza** [comment]
@abeyang this looks great -- should we perhaps consider updating this styling at a template level? So other IA's using the record template will be styled like this?

**Comment by abeyang** [comment]
@jagtalon The keys should be in title case ("Last Updated"), and can we see if we can remove the colons? It seems like the dev is able to do so for the [BPM Goodie](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/803#issuecomment-70484379).

**Comment by MrChrisW** [comment]
@abeyang That's because the Goodie is manually generating the html content and not using the template system like the Spice. :smile:

**Comment by jagtalon** [comment]
@abeyang I updated the "Last Updated"--the colons, however, are at the template level (internally) and can't be changed from here. I can add an Asana task to control that.

https://ddh1.duckduckgo.com/?q=whois+duckduckgo.com&ia=whois

**Comment by abeyang** [comment]
I see. Yes, let's put that as a task (removing colons), but otherwise, looks good! :+1:

**Comment by jagtalon** [comment]
@abeyang nice!

@MrChrisW 

oisodse

**Pull request merged**